bittide-instances/src/Bittide/Instances/Hitl/FincFdec.hs

```diff
@@ -168,40 +163,39 @@ goFincFdecTests clk rst clkControlled testSelect miso =
 
 fincFdecTests ::
   -- Pins from internal oscillator:
-  "CLK_125MHZ" ::: DiffClock Basic125 ->
+  "CLK_125MHZ" ::: DiffClock Ext125 ->
 
   -- Pins from clock board:
-  "USER_SMA_CLOCK" ::: DiffClock Basic200B ->
-  "MISO" ::: Signal Basic200A Bit -> -- SPI
+  "USER_SMA_CLOCK" ::: DiffClock Ext200 ->
+  "MISO" ::: Signal Basic200 Bit -> -- SPI
 
   "" :::
     ( "" :::
-      ( "done"    ::: Signal Basic200A Bool
-      , "success" ::: Signal Basic200A Bool
+      ( "done"    ::: Signal Basic200 Bool
+      , "success" ::: Signal Basic200 Bool
       )
 
     -- Freq increase / freq decrease request to clock board
     , "" :::
-      ( "FINC"    ::: Signal Basic200A Bool
-      , "FDEC"    ::: Signal Basic200A Bool
+      ( "FINC"    ::: Signal Basic200 Bool
+      , "FDEC"    ::: Signal Basic200 Bool
       )
 
       -- SPI to clock board:
     , "" :::
-      ( "SCLK" ::: Signal Basic200A Bool
-      , "MOSI" ::: Signal Basic200A Bit
-      , "CSB"  ::: Signal Basic200A Bool
+      ( "SCLK" ::: Signal Basic200 Bool
+      , "MOSI" ::: Signal Basic200 Bit
+      , "CSB"  ::: Signal Basic200 Bool
       )
     )
 fincFdecTests diffClk controlledDiffClock spiIn =
   ((testDone, testSuccess), fIncDec, spiOut)
  where
   clkControlled = ibufds controlledDiffClock
 
-  (clk, clkStable0) = clockWizardDifferential (SSymbol @"pll") diffClk noReset
-  clkStable1 = xpmCdcSingle clk clk clkStable0 -- improvised reset syncer
+  (clk, clkStableRst) = clockWizardDifferential diffClk noReset
+  clkStable1 = unsafeToActiveLow clkStableRst
```

Comment:
Are the outputs of `clockWizardDifferential` already synchronized here?

Reply:
I'd hope so, since the documentation states:

_This function incorporates [resetSynchronizer](https://hackage.haskell.org/package/clash-prelude-1.8.1/docs/Clash-Signal.html#v:resetSynchronizer)s to convert the locked output port into proper [Reset](https://hackage.haskell.org/package/clash-prelude-1.8.1/docs/Clash-Explicit-Signal.html#t:Reset) signals for the output domains which will keep the circuit in reset while the clock is still stabilizing._

Reply:
Yes, this has been safe as of the latest Clash release thanks to the work of @DigitalBrains1 !